web/viewer.html

```diff
@@ -78,7 +83,11 @@
 <!--#endif-->
 
 <!--#if !MINIFIED -->
+<!--#if MOZCENTRAL-->
+<!--<script src="resource://pdf.js/web/viewer.js"></script>-->
+<!--#else-->
 <!--<script src="viewer.js"></script>-->
+<!--#endif-->
 <!--#else-->
 <!--#include viewer-snippet-minified.html-->
 <!--#endif-->
```

Comment:
This is slightly hard to read, given the nested ifs. Can we please change it to the following instead?
```suggestion
<!--#if MINIFIED-->
<!--#include viewer-snippet-minified.html-->
<!--#elif MOZCENTRAL-->
<!--<script src="resource://pdf.js/web/viewer.js"></script>-->
<!--#else-->
<!--<script src="viewer.js"></script>-->
<!--#endif-->
```